content/models.py

```diff
@@ -15,7 +15,7 @@ def __str__(self):
 
 
 class Ourteam(models.Model):
-    img = models.ImageField(upload_to='uploads')
+    img = models.ImageField(blank=True, null=True)
```

Comment:
Its better to include a directory where files will be upload so `upload_to='uploads` should stay there